addons/packages/kapp-controller/0.41.2/package.yaml

```diff
@@ -12,7 +12,7 @@ spec:
     spec:
       fetch:
         - imgpkgBundle:
-            image: projects.registry.vmware.com/tce/kapp-controller@sha256:75d09b5b7c0e2996aec16f7725628b7e37ff978af5db60a6ee289cce322eb453
+            image: projects.registry.vmware.com/tce/kapp-controller@sha256:d32526854ff0c053b1025329dec1d6ff1f327f4e0beaab0cde28bbb80f4aa025
```

Comment:
I'm honestly not sure why this SHA changed, but happy to trust you.

Reply:
the SHA changed because an overlay and the values.yaml changed.